app/main/views/users.py

```diff
@@ -313,7 +315,7 @@ def export_users_for_framework(framework_slug):
             'framework_agreement': framework_agreement,
             'application_result': application_result,
             'variations_agreed': variations_agreed,
-            'published_service_count': supplier_id_published_service_count.get(sf.supplier_id, 0)
+            'published_service': bool(has_a_published_service)
```

Comment:
I know this isn't really anything to do with this PR, but can we add a note to this return that this is a non-standard format? For JSON responses from the API we generally use camelCase rather than underscoring. But would be good to call it out so that we know this is an odd case that (should but probably won't) get fixed one day.

Reply:
Done 👍